CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # [Versions](https://mui.com/versions/)
 
+## v6.0.0-alpha.3
+
+<!-- generated comparing v6.0.0-alpha.2..next -->
+
+_Apr 15, 2024_
+
+A big thanks to the 19 contributors who made this release possible.
+This release was mostly about supporting CSS extraction and API standardization.
```

Comment:
```suggestion
This release was mostly about converting components to Pigment CSS and API standardization.
```

I'd use the "Pigment CSS" brand name as much as possible (to really strengthen it) instead of what it does (CSS extraction, which won't make folks immediately think of Pigment).

Reply:
Good idea ⭐